doc/source/whatsnew/v2.1.2.rst

```diff
@@ -17,6 +17,7 @@ Fixed regressions
 - Fixed regression in :meth:`DataFrame.join` where result has missing values and dtype is arrow backed string (:issue:`55348`)
 - Fixed regression in :meth:`DataFrame.resample` which was extrapolating back to ``origin`` when ``origin`` was outside its bounds (:issue:`55064`)
 - Fixed regression in :meth:`DataFrame.sort_index` which was not sorting correctly when the index was a sliced :class:`MultiIndex` (:issue:`55379`)
+- Fixed performance regression in Copy-on-Write mechanism (:issue:`55256`, :issue:`55245`)
```

Comment:
The regression occurs without Copy-on-Write too. I think we should mention that here.

Reply:
Yeah I struggled a bit with the wording, any suggestions?

Reply:
Maybe
```
Fixed performance regression in DataFrame copying, DataFrame iteration, and groupby methods with user defined functions.
```
?

I think it's better to leave the copy-on-write part out - I personally couldn't find a way to word it without making it seem like the issue was with Copy-on-Write on only.

Reply:
Yeah I am not really happy with listing methods, since this affects all kinds of things with wide data frames